test/api/sse-dashboard.test.js

```diff
@@ -0,0 +1,313 @@
+          },
+          farmUtilisation: {
+            activeHours: 0,
+            failedHours: 0,
+            idleHours: 0,
+            offlineHours: 0,
+            activeHoursPercent: null,
+            failedHoursPercent: null,
+            idleHoursPercent: null,
+            offlineHoursPercent: null
+          },
+          printerHeatMaps: {
+            heatStatus: [],
+            heatProgress: [],
+            heatTemps: [],
+            heatUtilisation: []
+          },
+          utilisationGraph: {},
+          temperatureGraph: [
+            {
+              name: "Actual Tool",
+              data: [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            },
+            {
+              "name": "Target Tool",
+              "data": [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            },
+            {
+              "name": "Actual Bed",
+              "data": [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            },
+            {
+              "name": "Target Bed",
+              "data": [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            },
+            {
+              "name": "Actual Chamber",
+              "data": [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            },
+            {
+              "name": "Target Chamber",
+              "data": [
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                },
+                {
+                  x: expect.any(Number),
+                  y: expect.any(Number),
+                }
+              ]
+            }
+          ],
+          currentIAQ: null,
+          currentTemperature: null,
+          currentPressure: null,
+          currentHumidity: null,
+          enviromentalData: expect.any(Array)
+        },
+        dashboardSettings: {
+          defaultLayout: [
```

Comment:
I could call the getDefault from the constants provider file, this is also fine by me.